package.json

```diff
@@ -99,6 +99,7 @@
     "moment": "^2.22.1",
     "noty": "^3.2.0-beta",
     "openport": "0.0.5",
+    "qrcode": "^1.4.4",
```

Comment:
would be cool if we could add an image in the center. I don't think this module does that unfortunately.

Reply:
@tsingh777 Yes that would be so cool. :) Theoretically, we can, it's a canvas rendering, so it can be modified. I have visiting cards with such modified QR codes, and on some basic phones, it won't pick up the info. It actually depends upon the contrast of the image. If I can get a UI, I can give it a shot.